app/src/main/java/com/mmm/parq/fragments/DriverSettingsFragment.java

```diff
@@ -19,7 +23,25 @@
     private RecyclerView mRecyclerView;
     private ItemAdapter mItemAdapter;
 
-    public DriverSettingsFragment() {}
+    private View.OnClickListener mOnClickListener;
+
+    public DriverSettingsFragment() {
+        mOnClickListener = new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int itemPosition = mRecyclerView.getChildAdapterPosition(v);
+
+                // TODO(matt): figure out a better way to differentiate clicks; doesn't scale
+                switch (itemPosition) {
```

Comment:
I feel like then we are depending on raw text, instead of some underlying enum value (which I would like to do)